### PR TITLE
creating an endpoint to save teammember data

### PR DESCRIPTION
**Pull request by Aman16-ai**
## [Backend] Add the API for adding the Team member details in db

 Closes: #722

### 

- Created a model schema for Team members. 

- Created a route (http://localhost:3500/teamMember/addTeamMember) to save team members details in db. 

- Created a middleware fetchuser which gives the user details corresponding to a token.

- I used multer module to store the team member's image.

## Created a feature to save team member details 

_Put an `x` in the boxes that apply_

- [ x] Bugfix (non-breaking change which fixes an issue)
- [x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update (Documentation content changed)
- [ ] Other (please describe): 

## Checklist

_Put an `x` in the boxes that apply_

- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x ] My changes generate no new warnings
- [x ] My changes does not break the current system and it passes all the current test cases.

## Screensh
Screenshot (707)
Screenshot (708)
Screenshot (709)
Screenshot (710)
ots

Please attach the screenshots of the changes made in case of change in user interface

## Other information

Any other information that is important to this pull request

### CODE REVIEW

**Review by AyushSingh22 — APPROVED** [review]
Codes are simple & clean

File: backend/helpers/middlewares/auth.js [hunk 1]
```diff
@@ -37,5 +37,18 @@ const authMiddleware = async (req, res, next) => {
   res.locals.decode = payload;
   next();
 };
-
-module.exports = { authMiddleware, generateJWT, verifyToken };
+function FetchUser(req, res, next) {
```
Comment (Kajol-Kumari) [inline backend/helpers/middlewares/auth.js:40]: [question] Why did we need to introduce this function, why can't be use the `authMiddleware` defined above in this file?

File: backend/app/routes/teamMember/index.js [hunk 2]
```diff
@@ -0,0 +1,10 @@
+const router = require('express').Router({ mergeParams: true });
+const addTeam = require('./addTeam');
+
+const { FetchUser } = require('../../../helpers/middlewares/auth');
+const multer = require('multer');
+
+const upload = multer({ dest: 'uploads/teamMembersProfile' });
```
Comment (Kajol-Kumari) [inline backend/app/routes/teamMember/index.js:7]: Can we please think of other way without introducing a new folder `upload` for just storing the images?

**Review by Kajol-Kumari — CHANGES_REQUESTED** [review]
@Aman16-ai code looks fine, just need few clarifications.
Also, can you please remove the binary files from this pr as we don't need it

File: backend/helpers/middlewares/auth.js (hunk 1 above)
Comment (Aman16-ai) [reply backend/helpers/middlewares/auth.js:40]: I was not using bearer token. So i was getting an error. So i had created this function but now i have removed this fetchuser function and using authMiddleware.

Comment (Aman16-ai) [reply backend/helpers/middlewares/auth.js]: yes

**Review by Kajol-Kumari — APPROVED** [review]
Nice and Clean!
LGTM 👍

**Pull request merged**